buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -10,6 +10,7 @@ object Versions {
     val slf4jAPI = "1.0.0"
     val logbackClassic = "1.2.3"
     val axmlParser = "1.0"
+    val apkSig = "8.5.0"
```

Comment:
When Marathon plugin gets loaded, `apksig` dependency might be already loaded into memory, because it's a transitive dependency of AGP.
We should use the same version as AGP to avoid running against a lower version than we compile against.